src/java/com/zimbra/oauth/handlers/impl/GoogleOAuth2Handler.java

```diff
@@ -85,6 +86,16 @@
          */
         protected static final String AUTHENTICATE_URI = "https://www.googleapis.com/oauth2/v4/token";
 
+        /**
+         * The contacts endpoint for Google.
+         */
+        protected static final String CONTACTS_URI = "https://people.googleapis.com/v1/people/me/connections?personFields=names,emailAddresses,organizations,phoneNumbers,addresses,events,birthdays,biographies,nicknames,urls,photos,userDefined,skills,interests,braggingRights,relationshipInterests,relationshipStatuses,occupations,taglines";
```

Comment:
Do you think this can change and should go to local config

Reply:
The base url won't change unless the api does, which may require us to make changes to the code anyway.

Those properties are the ones we support parsing; making it dynamic wouldn't achieve anything if we don't support a new property, but it could allow limiting which ones are imported. This would be custom to google, though, unless we add this as a feature for all importers (outside the scope of this ticket).

Reply:
Ok sounds good